msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/ManagedIdentityResponse.java

```diff
@@ -9,15 +9,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.time.Instant;
-import java.time.format.DateTimeFormatter;
-import java.time.format.DateTimeParseException;
-import java.util.Locale;
-
 @Getter
 public class ManagedIdentityResponse {
```

Comment:
Should not be public. I think @neha-bhargava already fixed this, so just a heads up when merging.